packages/rainbowkit/src/components/ConnectOptions/ConnectDetails.tsx

```diff
@@ -1,45 +1,46 @@
+import { getGradientRGBAs } from "../../utils/colors";
+import { InstructionStepName } from "../../wallets/Wallet";
 import {
   WalletConnector,
   useWalletConnectors,
-} from '../../wallets/useWalletConnectors';
-import { AsyncImage } from '../AsyncImage/AsyncImage';
-import { loadImages } from '../AsyncImage/useAsyncImage';
-import { Box, BoxProps } from '../Box/Box';
-import { ActionButton } from '../Button/ActionButton';
-import { CreateIcon, preloadCreateIcon } from '../Icons/Create';
-import { RefreshIcon, preloadRefreshIcon } from '../Icons/Refresh';
-import { ScanIcon, preloadScanIcon } from '../Icons/Scan';
-import { SpinnerIcon } from '../Icons/Spinner';
-import { QRCode } from '../QRCode/QRCode';
-import { ModalSizeContext } from '../RainbowKitProvider/ModalSizeContext';
-import { Text } from '../Text/Text';
-import { WalletStep } from './DesktopOptions';
+} from "../../wallets/useWalletConnectors";
+import { AsyncImage } from "../AsyncImage/AsyncImage";
+import { loadImages } from "../AsyncImage/useAsyncImage";
+import { Box, BoxProps } from "../Box/Box";
+import { ActionButton } from "../Button/ActionButton";
+import { CreateIcon, preloadCreateIcon } from "../Icons/Create";
+import { RefreshIcon, preloadRefreshIcon } from "../Icons/Refresh";
+import { ScanIcon, preloadScanIcon } from "../Icons/Scan";
+import { SpinnerIcon } from "../Icons/Spinner";
+import { QRCode } from "../QRCode/QRCode";
+import { I18nContext } from "../RainbowKitProvider/I18nContext";
+import { ModalSizeContext } from "../RainbowKitProvider/ModalSizeContext";
+import { Text } from "../Text/Text";
+import { WalletStep } from "./DesktopOptions";
 
 const getBrowserSrc: () => Promise<string> = async () => {
   const browser = getBrowser();
   switch (browser) {
     case BrowserType.Arc:
-      return (await import('../Icons/Arc.svg')).default;
+      return (await import("../Icons/Arc.svg")).default;
     case BrowserType.Brave:
-      return (await import('../Icons/Brave.svg')).default;
+      return (await import("../Icons/Brave.svg")).default;
     case BrowserType.Chrome:
-      return (await import('../Icons/Chrome.svg')).default;
+      return (await import("../Icons/Chrome.svg")).default;
     case BrowserType.Edge:
-      return (await import('../Icons/Edge.svg')).default;
+      return (await import("../Icons/Edge.svg")).default;
     case BrowserType.Firefox:
-      return (await import('../Icons/Firefox.svg')).default;
+      return (await import("../Icons/Firefox.svg")).default;
     case BrowserType.Opera:
-      return (await import('../Icons/Opera.svg')).default;
+      return (await import("../Icons/Opera.svg")).default;
     case BrowserType.Safari:
-      return (await import('../Icons/Safari.svg')).default;
+      return (await import("../Icons/Safari.svg")).default;
     default:
-      return (await import('../Icons/Browser.svg')).default;
+      return (await import("../Icons/Browser.svg")).default;
   }
```

Comment:
@KosmosKey It seems like you might have a local linter problem; try disabling any eslint plugins and enable this: https://marketplace.visualstudio.com/items?itemName=biomejs.biome

Reply:
Will fix that 🙏